caster-front/src/assets/elementplus.scss

```diff
@@ -326,4 +340,4 @@
 
 .el-table .descending .sort-caret.descending {
     border-top-color: #{$black};
-}
+}
```

Comment:
normally the CI should fail as pre-commit forces a blank line at the ending of a file.
Will take a look at this in a new issue.

Reply:
Thanks